medicinal_plants/lib/src/screens/profile-screen/widget/buttons_screen.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/cupertino.dart';
+
+import '../../../common/buttons/custom_button.dart';
+
+class ProfileButtonsScreen extends StatelessWidget {
+	const ProfileButtonsScreen({super.key});
+
+	@override
+	Widget build(BuildContext context) {
+		double width = MediaQuery.of(context).size.width;
+		double height = MediaQuery.of(context).size.height;
+		
+		return Column(
+			children: [
+				CustomButton(
+					onPressed: () {
+						// Add your Personal Information button logic here
```

Comment:
I dont think this is the right way of implementation on this,
We will refactor this soon